docs/src/main/diataxis/doc-contribute-quarkus-docs-howto.adoc

```diff
@@ -0,0 +1,76 @@
+////
+This document is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="howto-contribute-quarkus-docs"]
+= Contribute to Quarkus documentation
+include::attributes.adoc[]
+:toc: preamble
+:asciidoc: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/
+:quarkus-docs: https://github.com/quarkusio/quarkus/tree/main/docs
+:fn-diataxis: footnote:diataxis[Procida, D. Diátaxis documentation framework. https://diataxis.fr/]
+
+This guide shows you the recommended steps for making successful contributions to Quarkus documentation.
+
+== Prerequisites
+
+Quarkus docs are built from source written using {asciidoc}[Asciidoc], a lightweight markup language.
+
+We suggest you have the following materials nearby:
+
+- An editor or IDE that provides syntax highlighting and previews for asciidoc, either natively or using a plugin.
+- An https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/[Asciidoc syntax reference]
+- The xref:{docref}/doc-reference.adoc[Quarkus documentation reference] for required syntax and other conventions.
+
+== Sources
+
+- Documentation for Quarkus core and most extensions is located in the `docs` directory of the {quarkus-docs}[Quarkus GitHub repository].
+- Docs for Quarkiverse or other third-party extensions are pulled directly from those repositories.
+
+== Building and previewing Quarkus documentation
+
+Asciidoc syntax highlighting and the preview provided by an IDE can be sufficient for minor documentation changes.
+For significant changes or any changes related to Quarkus configuration documentation,
+we recommend that you run the build and view the resulting output before submitting your changes for review.
+
+include::{includes}/compile-quarkus-quickly.adoc[tag=quickly-install-docs]
+
+This will produce a few things:
+
+- Generated configuration properties documentation will be located in the  `target/asciidoc/generated/config/` directory.
+- Asciidoc output (html files) will be found in the `docs/target/generated-docs/` directory.
+
+As you make additional changes, you can build the `docs` module specifically to update the generated HTML:
+
+[source,shell]
+----
+$ ./mvnw -f docs clean install
+----
+
+If you change the configuration properties of an extension,
+rebuild that extension first (to regenerate the associated config docs in `target/asciidoc/generated/config/`),
+and then rebuild the `docs` module.
+
+== Creating pull requests for doc updates
+
+:gh-pull-requests-fork: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork
+:gh-about-forks: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks
```

Comment:
I fail to see how it's an improvement to do that when you only use the link once?

Reply:
because it's really really long. Makes the sentence easier to read with less garbage link baloney in it.

Reply:
My guess is that it will end up being  bad idea if we generalize that but I suppose we will see that soon enough.

Reply:
It's a key asciidoc capability (markdown has similar), and in this case, I kept the links close to where they are used. But it makes the text easier to comprehend (without sifting through long links)